src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -77,6 +79,7 @@ const LibraryAuthoringPage = () => {
   const handleTabChange = (key: string) => {
     setTabKey(key);
     navigate(key);
+    setSearchParams(searchParams);
   };
```

Comment:
```suggestion
  const handleTabChange = (key: string) => {
    setTabKey(key);
    navigate({
      pathname: key,
      search: searchParams.toString(),
    });
  };
```

Reply:
@pomegranited I clicked post accidentally before writing my comment 😅 . The tests were failing seemingly because the search params being updated later was causing issues with the location state for some reason. Putting them inside the `navigate` as one operation fixes the issue, and maintains the functionality we want (thanks @rpenido for the help!).

Reply:
Thank you so much @yusuf-musleh and @rpenido ! That test failure kept me awake last night, I couldn't figure it out!